examples/kosmos-2-patch14-224/serve.yaml

```diff
@@ -0,0 +1,15 @@
+images:
+  kosmos-2-patch14-224-gpu:
+    base: autonomi/nos:0.1.0rc2-gpu
+    pip:
+      - torch==2.0.1
+      - git+https://github.com/huggingface/transformers
```

Comment:
`transformers>4.35.0 `should be sufficient enough, or do we need some featuer from the main branch?

Reply:
Fixed. Although some people prefer to use the most up-to-date version of hf transformers so that the examples has more usability as a starting point/building block as the eventual users can add on top of it more easilly with new features down the line.